Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -1177,6 +1177,14 @@ theorem deleteVerts_inter_verts_set_right_eq :
     G'.deleteVerts (s ∩ G'.verts) = G'.deleteVerts s := by
   ext <;> simp +contextual [imp_false]
 
+lemma disjoint_deleteVerts_verts_image_val {s : Set V}
+    {t : Set ((⊤ : Subgraph G).deleteVerts s).verts} :
+    Disjoint s (Subtype.val '' t) := by
```

Comment:
This lemma looks a bit strange to me, and you don't use it in this PR - can we remove it for now?

Reply:
Done:)